variants/adafruit_itsybitsy/pins_arduino.h

```diff
@@ -40,4 +40,16 @@
 #define SPI_HOWMANY    (1u)
 #define WIRE_HOWMANY   (2u)
 
+// D pins
+#define D2             (12u)
+#define D3             (5u)
+#define D4             (4u)
+#define D5             (14u)
+#define D7             (6u)
+#define D9             (7u)
+#define D10            (8u)
+#define D11            (9u)
+#define D12            (10u)
+#define D13            (11u)
+
```

Comment:
Also looks like D24-D25 are included per the schematic here https://cdn-learn.adafruit.com/assets/assets/000/101/925/original/adafruit_products_ItsyRP_sch.png?1619632724

Reply:
D24 and D25 are mapped correctly
https://github.com/adafruit/Adafruit-ItsyBitsy-RP2040-PCB/blob/main/Adafruit%20ItsyBitsy%20RP2040%20pinout.svg

Reply:
thanks!